### DISCUSSION TITLE
support end date for NCalc 1.3.8

### DISCUSSION THREAD

**Discussion by Aishbeck** [ncalc/ncalc#339] [General]
I am looking for Ncalc 1.3.8 support end date or year.

**Comment by Bykiev** [comment]
Hi, can you please describe in more details your issue? What does 'support end date or year' means?

**Reply by Aishbeck** [reply]
I am documenting end of support date for the components used by our product.
so Ncalc 1.3.8 is something we use,  I need the data till how long this specific version was supported or is still in support, example fixes needed, issues addressed, security updates

**Reply by Bykiev** [reply]
I think this version is not supported anymore (I believe you're talking about this package https://www.nuget.org/packages/ncalc). The best choice will be upgrade and use the latest version from this repo.

**Reply by Aishbeck** [reply]
yes, we are planning to upgrade, but for documentation reasons, I am still looking when the support got ended.

**Reply by Bykiev** [reply]
Unfortunately, it's not known. just take this version release date
